apm-sniffer/config/agent.config

```diff
@@ -15,31 +15,31 @@
 # limitations under the License.
 
 # The agent namespace
-# agent.namespace=default-namespace
+# agent.namespace=${SW_AGENT_NAMESPACE:default-namespace}
 
 # The service name in UI
-agent.service_name=Your_ApplicationName
+agent.service_name=${SW_AGENT_NAME:Your_ApplicationName}
 
 # The number of sampled traces per 3 seconds
 # Negative number means sample traces as many as possible, most likely 100%
-# agent.sample_n_per_3_secs=-1
+# agent.sample_n_per_3_secs=${SW_AGENT_SAMPLE:-1}
 
 # Authentication active is based on backend setting, see application.yml for more details.
-# agent.authentication = xxxx
+# agent.authentication = ${SW_AGENT_AUTHENTICATION:xxxx}
 
 # The max amount of spans in a single segment.
 # Through this config item, skywalking keep your application memory cost estimated.
-# agent.span_limit_per_segment=300
+# agent.span_limit_per_segment=${SW_AGENT_SPAN_LIMIT:300}
 
 # Ignore the segments if their operation names start with these suffix.
-# agent.ignore_suffix=.jpg,.jpeg,.js,.css,.png,.bmp,.gif,.ico,.mp3,.mp4,.html,.svg
+# agent.ignore_suffix=${SW_AGENT_IGNORE_SUFFIX:.jpg,.jpeg,.js,.css,.png,.bmp,.gif,.ico,.mp3,.mp4,.html,.svg}
 
 # If true, skywalking agent will save all instrumented classes files in `/debugging` folder.
 # Skywalking team may ask for these files in order to resolve compatible problem.
-# agent.is_open_debugging_class = true
+# agent.is_open_debugging_class = ${SW_AGENT_OPEN_DEBUG:true}
 
 # Backend service addresses.
-collector.backend_service=127.0.0.1:11800
+collector.backend_service=${SW_AGENT_COLLECTOR_BACKEND_SERVICES:127.0.0.1:11800}
 
 # Logging level
-logging.level=DEBUG
+logging.level=${SW_LOGGING_LEVEL:INFO}
```

Comment:
Let's keep this in `DEBUG` level to help the new users see more info. OK?

Reply:
ok, I just think it's better to use `INFO` when package, in a way. :)